src/cta/index.js

```diff
@@ -77,11 +77,23 @@ button.addEventListener('click', function(event) {
     }
 });
 
+button.addEventListener('keypress', function (event) {
```

Comment:
Since we're allowing users to "tab" into buttons and links via native <button> and <a>, these elements need to be able to perform expected action when a user presses "enter". I noticed that somewhere else in the code we do this by simply calling "click" event which then handles the action. I did the same thing here.